examples/all_the_things.py

```diff
@@ -32,9 +33,15 @@
 #from openhtf.io.output import mfg_inspector
 
 
+conf.Declare('foo', default_value='bar',
+             description='Example of conf value.')
+
+
 @plug(example=example_plug.ExamplePlug)
-def example_monitor(example):
-  return example.Increment()
+def example_monitor(self, example):
```

Comment:
don't call it 'self', 'self' very specifically is intended for class methods to refer to the bound instance.  i think we typically call it 'test', but really anything that's not a word that already means something else.